container-di/src/main/java/com/yahoo/container/di/Osgi.java

```diff
@@ -25,6 +25,10 @@ default BundleClasses getBundleClasses(ComponentSpecification bundle, Set<String
         return new BundleClasses(new MockBundle(), Collections.emptySet());
     }
 
+    default void installPlatformBundles(Collection<FileReference> bundles) {
```

Comment:
The interface also behaves as a mock implementation of itself? Never seen that before.

Reply:
Perhaps it made sense in Scala :)